content/reference/user/dedicated_ips/index.md

```diff
@@ -1,38 +1,38 @@
 ---
-title: "Dedicated IPs"
-description: "Dedicated IP addresses for your pools"
-lead: "Dedicated IP addresses for your pools"
+title: "Dedicated IP Addresses"
+description: "Dedicated IP addresses for your locations"
+lead: "Dedicated IP addresses for your locations"
 date: 2021-03-10T14:29:04+00:00
 lastmod: 2021-08-05T13:13:30+00:00
 weight: 21070
 ---
 
-Dedicated IPs allow you to have control over the injectors' IP addresses.
+Dedicated IP Addresses allow you to have control over the load generators' IP addresses.
 This is useful, for example, if your target system performs some sort of IP address filtering.
 
 {{< img src="dedicated-ips.png" alt="Dedicated IPs" caption="Dedicated IPs" >}}
 
 ## Managing
 
-To access the dedicated IPs section, click on **Dedicated IPs** in the navigation bar.
+To access the dedicated IP addresses section, click on **Dedicated IP Addresses** in the navigation bar.
 
 {{< alert tip >}}
-You can request dedicated IPs through [technical support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8/group/12/create/59).
+You can request dedicated IP addresses through [technical support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8/group/12/create/59).
 
 Please provide:
 - Organization SLUG
-- Desired number of dedicated IPs per region
+- Desired number of dedicated IP addresses per location
 - Contact Email
 - GitHub username
 A sales person will contact you.
 {{< /alert >}}
 
-The Dedicated IPs table shows your available dedicated IP addresses. Each one belongs to a specific pool.
+The Dedicated IP addresses table shows your available dedicated IP addresses. Each one belongs to a specific location.
 
 ## Usage
 
-You can enable the use of dedicated IPs when [configuring simulation pools]({{< ref "../simulations#step-3-pools-configuration" >}}).
+You can enable the use of dedicated IP addresses when [configuring simulation locations]({{< ref "../simulations#step-3-locations-configuration" >}}).
 
-When starting a run of a simulation configured to use dedicated IPs,
-if you have enough dedicated IPs available to satisfy the size of the configured pools,
-they will be reserved for the run duration.  If you don’t have enough dedicated IPs available, the run won't start.
+When starting a run of a simulation configured to use dedicated IP addresses,
+if you have enough dedicated IP addresses available to satisfy the size of the configured locations,
+they will be reserved for the run duration.  If you don’t have enough dedicated IP addresses available, the run won't start.
```

Comment:
documentation.png still has a margin issue.

Reply:
credits-consumption.png is still missing the legends

Reply:
overview.png is supposed to display the original landing page with the warning message when there's no package nor simulation.

Reply:
* documentation.png => Thank @franciscotco!
* credits-consumption.png => bug fixed but not deployed (even on dev environment)
* overview.png => created another organization to have empty data